js/src/logger.ts

```diff
@@ -1143,6 +1252,15 @@ export class Logger<IsAsyncFlush extends boolean> implements Exportable {
     });
   }
 
+  // TODO: do we want convenience methods?
```

Comment:
Personally feels unnecessary but not a strong opinion.

Reply:
Removed